data/assets/scene/solarsystem/sun/habitablezone.asset

```diff
@@ -0,0 +1,34 @@
+local assetHelper = asset.require("util/asset_helper")
+local transforms = asset.require("./transforms")
+local textures = asset.require('scene/milkyway/habitable_zones/habitable_zone_textures').TexturesPath
+
+local HabitableZone = {
+    Identifier = "SunHabitableZone",
+    Parent =  transforms.SunECLIPJ2000.Identifier,
+    Renderable = {
+        Type = "RenderableHabitableZone",
+        Enabled = false,
+        Texture = textures .. "/hot_to_cold_faded.png",
+        EffectiveTemperature = 5780, -- Kelvin
+        Luminosity = 1, -- solar
+        Opacity = 0.1
+    },
+    GUI = {
+        Name = "Sun Habitable Zone",
+        Path = "/Solar System/Sun",
+        Description = "Habitable zone for the sun in our solar system."
+    }
+}
+
+assetHelper.registerSceneGraphNodesAndExport(asset, { HabitableZone })
+
+asset.meta = {
+    Name = "Sun Habitable Zone",
+    Version = "1.0",
+    Description = [[ The habitable zone around our sun, computed using formula and 
+    coefficients by Kopparapu et al. (2015) https://arxiv.org/abs/1404.5292]],
```

Comment:
@micahnyc does this look ok? Or should the Url to the paper be in`URL` part of the `asset.meta`?

Reply:
This is good. A lot of the DU stuff has the source in the description so it will be consistent with what we have.